contrib/create_char.py

```diff
@@ -0,0 +1,203 @@
+
```

Comment:
thanks! remove the first line?
Also can you put it under a scripts subfolder?

Reply:
done.